contracts/connection-router/tests/test.rs

```diff
@@ -131,1335 +133,1335 @@ fn route() {
     // try to route twice
     let res = config.app.execute_contract(
         eth.gateway.clone(),
-        config.contract_address.clone(),
+        config.connection_router.contract_addr.clone(),
         &ExecuteMsg::RouteMessages(msgs.clone()),
         &[],
     );
```

Comment:
why did you not replace this with a call to the router itself?

Reply:
weird. because I already have the same piece of code in this part:

```
let res = config.app.execute_contract(
        eth.gateway.clone(),
        config.connection_router.contract_addr.clone(),
        &ExecuteMsg::RouteMessages(msgs.clone()),
        &[],
    );
```
marking this as resolved..